panel/pane.py

```diff
@@ -560,6 +560,74 @@ def _img(self):
         return b.getvalue()
 
 
+def latex_to_img(text, size=25):
+    """
+    Returns PIL image for LaTeX equation text, using matplotlib's rendering.
+    Usage: latex_to_img(r'$\frac(x}{y^2}$')
+    From https://stackoverflow.com/questions/1381741.
+    """
+    import matplotlib.pyplot as plt
+    from PIL import Image, ImageChops
+    import io
+    
+    buf = io.BytesIO()
+    plt.rc('text', usetex=True)
+    plt.rc('font', family='serif')
+    plt.axis('off')
```

Comment:
Should not use global figure, doesn't need to use savefig and should also make dpi configurable:

```python
fig = plt.figure()
ax = fig.add_subplot(111)
ax.axis('off')
ax.text(0.05, 0.5, '{text}'.format(text=text), size=size)
fig.set_dpi(dpi)
fig.canvas.print_figure(b)
plt.close(fig)
```

Reply:
Thanks; fixed.

Reply:
Do you have any suggestions for how `dpi` and `size` relate in this code?  I'd like the docstring to describe how to use them, but I can't detect the practical difference between them.

Reply:
Have a look at the ``Matplotlib`` pane, it computes the size of the plot relative to a fixed dpi. Which means the dpi scales the resolution while the size will scale the actual size of plot on the page.

Reply:
I still can't tell the difference:

```
import panel as pn
from panel.pane import LaTeX
e = r'$\frac{x}{y^2}$'

pn.Row(LaTeX(e, size=25, dpi=100), LaTeX(e, size=50, dpi=100), LaTeX(e, size=25, dpi=200))
```
![image](https://user-images.githubusercontent.com/1695496/46685230-980acb80-cbba-11e8-8512-ec4efe53bc40.png)

Reply:
I can't tell where in the code you're scaling with a fixed dpi.

Reply:
I would have expected something like:

```
    def _imgshape(self, data):
        """Calculate and return image width,height"""
        w, h = super(LaTeX, self)._imgshape(data)
        w, h = (w/self.dpi), (h/self.dpi)
        return int(w*72), int(h*72)
```

Haven't tested it though.

Reply:
Seems to work:

![screen shot 2018-10-09 at 6 04 41 pm](https://user-images.githubusercontent.com/1550771/46685622-d7064480-cbed-11e8-9474-533fe5e5756f.png)

Reply:
Any idea why they sometimes overlap and have too much padding at other times?

Reply:
Thanks; I'll try it.  I haven't been able to tell how the spacing works out, here or in Panes in general. :-(

Reply:
The spacing is very weird:

```
import panel as pn
from panel.pane import LaTeX
e = r'$\frac{x}{y^2}$'

pn.Column(
    pn.Row(LaTeX(e), LaTeX(e)),
    pn.Row(LaTeX(e), "txt", LaTeX(e)))
```
![image](https://user-images.githubusercontent.com/1695496/46686466-fd13f080-cbbd-11e8-8ea0-a4477f078dde.png)

Do you have any idea what's going on there?

Reply:
(Code required for the above just now pushed to the branch)

Reply:
This particular spacing issue doesn't seem to be specific to LaTeX:

![image](https://user-images.githubusercontent.com/1695496/46687494-9f34d800-cbc0-11e8-92b6-604b67cbf71b.png)

Reply:
So I'll ignore that issue for this PR.